src/integration/react.js

```diff
@@ -24,7 +24,7 @@ export class PersistGate extends PureComponent<Props, State> {
   }
   _unsubscribe: ?Function
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
componentWillMount is being deprecated I think? is there another way to achieve the same effect?